intern.json

```diff
@@ -88,24 +87,21 @@
 			]
 		},
 
-		"fast": {
-			"description": "For running unit tests quickly",
-			"excludeInstrumentation": true,
+		"progress": {
+			"description": "For showing test progress",
 			"node": {
-				"reporters": { "name": "runner", "options": { "hideSkipped": true, "hidePassed": true } }
+				"reporters": { "name": "runner", "options": { "hideSkipped": true, "hidePassed": false } }
 			}
 		},
 
 		"coverage": {
 			"description": "Generate full coverage",
```

Comment:
We should put back the `extends` and `benchmark` properties, or add them to CI. I just commented them out to make dev faster.